### PR TITLE
feat: Added Greek language

### PR DESCRIPTION
**Pull request by Likecinema**
## Description
Added Greek Language

### CODE REVIEW

File: src/shared/locales/index.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ export const availableLanguages = [
     value: 'ca',
     label: 'Català'
   },
+  {
+    value: 'el',
```
Comment (agalwood) [inline src/shared/locales/index.js:18]: Please let it be in alphabetical order

**Review by agalwood — CHANGES_REQUESTED** [review]
Please adjust the code : )

**Comment by Likecinema** [comment]
Code adjusted, thanks for letting me know where the issue was :)

**Review by agalwood — APPROVED** [review]

**Comment by agalwood** [comment]
Thanks

**Pull request merged**